test/source/tests/compose.ts

```diff
@@ -269,25 +270,37 @@ export const defineComposeTests = (testVariant: TestVariant, testWithBrowser: Te
     }));
 
     ava.default('compose - settings - manually copied pubkey', testWithBrowser('ci.tests.gmail', async (t, browser) => {
+      const dbPage = await browser.newPage(t, TestUrls.extension('chrome/dev/ci_unit_test.htm'));
+      // add a contact containing 2 pubkeys to the storage
+      await dbPage.page.evaluate(async (pubkeys: string[]) => {
+        for (const pubkey of pubkeys) {
+          const key = await (window as any).KeyUtil.parse(pubkey);
+          await (window as any).ContactStore.update(undefined, 'tocopyfrom@example.test', { pubkey: key });
+        }
+      }, [testConstants.abcddfTestComPubkey, testConstants.abcdefTestComPubkey]);
       const inboxPage = await browser.newPage(t, TestUrls.extensionInbox('ci.tests.gmail@flowcrypt.test'));
-      let composeFrame = await InboxPageRecipe.openAndGetComposeFrame(inboxPage);
-      await ComposePageRecipe.fillMsg(composeFrame, { to: 'human@flowcrypt.com' }, 'just to load - will close this page');
-      await Util.sleep(2); // todo: should wait until actually loaded
-      await composeFrame.waitAndClick('@action-close-new-message');
-      await inboxPage.waitTillGone('@container-new-message');
-      composeFrame = await InboxPageRecipe.openAndGetComposeFrame(inboxPage);
-      await ComposePageRecipe.fillMsg(composeFrame, { to: 'human+manualcopypgp@flowcrypt.com' }, 'manual copied key');
+      const composeFrame = await InboxPageRecipe.openAndGetComposeFrame(inboxPage);
+      await ComposePageRecipe.fillMsg(composeFrame, { to: 'manualcopypgp@flowcrypt.com' }, 'manual copied key');
       await composeFrame.waitAndClick('@action-open-add-pubkey-dialog', { delay: 1 });
       await inboxPage.waitAll('@dialog-add-pubkey');
       const addPubkeyDialog = await inboxPage.getFrame(['add_pubkey.htm']);
       await addPubkeyDialog.waitAll('@input-select-copy-from');
       await Util.sleep(1);
-      await addPubkeyDialog.selectOption('@input-select-copy-from', 'human@flowcrypt.com');
+      await addPubkeyDialog.selectOption('@input-select-copy-from', 'tocopyfrom@example.test');
```

Comment:
@IvanPizhenko I added filling the contact store source contact with pubkeys 
and testing the resulting contact in the contact store via `dbPage`.
It would also be good to test that `@input-pubkey` textarea is visible before selecting the option and invisible right after.
And I guess "Manually importing Public Keys can be dangerous" only applies by pasting a pubkey text, but that's more a question for @tomholub

Reply:
@rrrooommmaaa I've updated test to check appearance/disapperance of the `@input-pubkey `.